core/app/services/spree/import_service/products/serializers/variant_attributes_serializer.rb

```
@@ -0,0 +1,29 @@
+      module Serializers
+        class VariantAttributesSerializer
+          VARIANT_ATTRIBUTES = %i[sku].freeze
+          NUMERIC_ATTRIBUTES = %i[cost_price weight height width depth].freeze
+
+          def initialize(row:, product_id:)
+            @row = row.symbolize_keys!
+            @product_id = product_id
+          end
+
+          def to_h
+            to_h ||= row.slice(*VARIANT_ATTRIBUTES).merge!(converted_attributes).merge(product_id: product_id)
```

RuboCop (1.75.5)

[warning] Useless assignment to variable - `to_h`. Use `||` instead of `||=`.

(Lint/UselessAssignment)